mllib/src/test/scala/org/apache/spark/ml/classification/LinearSVCSuite.scala

```diff
@@ -127,6 +127,14 @@ class LinearSVCSuite extends SparkFunSuite with MLlibTestSparkContext with Defau
     MLTestingUtils.checkCopyAndUids(lsvc, model)
   }
 
+  test("LinearSVC threshold can be any real value") {
```

Comment:
It would be nice to see a test that checks behavior of threshold while we're here. e.g. positive infinity predicts only class 0, negative infinity predicts only class 1.